Flaky test_no_markseen_in_team_profile (2.56.0-dev)

Log from CI:
```
=================================== FAILURES ===================================
___________________ test_no_markseen_in_team_profile[False] ____________________
[gw5] darwin -- Python 3.14.6 /Users/runner/work/core/core/deltachat-rpc-client/.tox/py/bin/python

team_profile = False
acfactory = <deltachat_rpc_client.pytestplugin.ACFactory object at 0x10795db20>

    @pytest.mark.parametrize("team_profile", [True, False])
    def test_no_markseen_in_team_profile(team_profile, acfactory):
        """
        Test that seen status is synchronized iff `team_profile` isn't set.
        """
        alice, bob = acfactory.get_online_accounts(2)
        if team_profile:
            bob.set_config("team_profile", "1")
    
        # Bob sets up a second device.
        bob2 = bob.clone()
        bob2.start_io()
    
        alice_chat_bob = alice.create_chat(bob)
        bob_chat_alice = bob.create_chat(alice)
        bob2.create_chat(alice)
        alice_chat_bob.send_text("Hello Bob!")
    
        message = bob.wait_for_incoming_msg()
        message2 = bob2.wait_for_incoming_msg()
        assert message2.get_snapshot().state == MessageState.IN_FRESH
    
        message.mark_seen()
    
        # Send a message and wait until it arrives
        # in order to wait until Bob2 gets the markseen message.
        # This also tests that outgoing messages
        # don't mark preceeding messages as seen in team profiles.
        bob_chat_alice.send_text("Outgoing message")
        while True:
            outgoing = bob2.wait_for_msg(EventType.MSGS_CHANGED)
            if outgoing.id != 0:
                break
        assert outgoing.get_snapshot().text == "Outgoing message"
    
        if team_profile:
            assert message2.get_snapshot().state == MessageState.IN_FRESH
        else:
>           assert message2.get_snapshot().state == MessageState.IN_SEEN
E           AssertionError: assert 13 == <MessageState.IN_SEEN: 16>
E            +  where 13 = {'chat_id': 12, 'dimensions_height': 0, 'dimensions_width': 0, 'download_state': 'Done', 'duration': 0, 'error': None, 'file': None, 'file_bytes': 0, 'file_mime': None, 'file_name': None, 'from_id': 10, 'has_deviating_timestamp': False, 'has_html': False, 'has_location': False, 'id': 13, 'info_contact_id': None, 'is_bot': False, 'is_edited': False, 'is_forwarded': False, 'is_info': False, 'original_msg_id': None, 'override_sender_name': None, 'parent_id': None, 'quote': None, 'reactions': None, 'received_timestamp': 1784651943, 'saved_message_id': None, 'sender': Contact(account=Account(manager=<deltachat_rpc_client.deltachat.DeltaChat object at 0x10795ce20>, id=3), id=10), 'show_padlock': True, 'sort_timestamp': 1784651942, 'state': 13, 'subject': 'Message from fbkcsyj5r@ci-chatmail.testrun.org', 'system_message_type': 'Unknown', 'text': 'Hello Bob!', 'timestamp': 1784651942, 'vcard_contact': None, 'view_type': 'Text', 'webxdc_href': None, 'chat': Chat(account=Account(manager=<deltachat_rpc_client.deltachat.DeltaChat object at 0x10795ce20>, id=3), id=12), 'message': Message(account=Account(manager=<deltachat_rpc_client.deltachat.DeltaChat object at 0x10795ce20>, id=3), id=13)}.state
E            +    where {'chat_id': 12, 'dimensions_height': 0, 'dimensions_width': 0, 'download_state': 'Done', 'duration': 0, 'error': None, 'file': None, 'file_bytes': 0, 'file_mime': None, 'file_name': None, 'from_id': 10, 'has_deviating_timestamp': False, 'has_html': False, 'has_location': False, 'id': 13, 'info_contact_id': None, 'is_bot': False, 'is_edited': False, 'is_forwarded': False, 'is_info': False, 'original_msg_id': None, 'override_sender_name': None, 'parent_id': None, 'quote': None, 'reactions': None, 'received_timestamp': 1784651943, 'saved_message_id': None, 'sender': Contact(account=Account(manager=<deltachat_rpc_client.deltachat.DeltaChat object at 0x10795ce20>, id=3), id=10), 'show_padlock': True, 'sort_timestamp': 1784651942, 'state': 13, 'subject': 'Message from fbkcsyj5r@ci-chatmail.testrun.org', 'system_message_type': 'Unknown', 'text': 'Hello Bob!', 'timestamp': 1784651942, 'vcard_contact': None, 'view_type': 'Text', 'webxdc_href': None, 'chat': Chat(account=Account(manager=<deltachat_rpc_client.deltachat.DeltaChat object at 0x10795ce20>, id=3), id=12), 'message': Message(account=Account(manager=<deltachat_rpc_client.deltachat.DeltaChat object at 0x10795ce20>, id=3), id=13)} = get_snapshot()
E            +      where get_snapshot = Message(account=Account(manager=<deltachat_rpc_client.deltachat.DeltaChat object at 0x10795ce20>, id=3), id=13).get_snapshot
E            +  and   <MessageState.IN_SEEN: 16> = MessageState.IN_SEEN

tests/test_something.py:1042: AssertionError
------------------------------ Captured log setup ------------------------------
INFO     root:rpc.py:129 RPC server ready. Core version: v2.56.0-dev

```

--- a/deltachat-rpc-client/tests/test_something.py
+++ b/deltachat-rpc-client/tests/test_something.py
@@ -1003,6 +1003,11 @@ def test_no_markseen_in_team_profile(team_profile, acfactory):
 
     message.mark_seen()
 
+    # The MDN is queued in `smtp_mdns`, which is drained only after the regular
+    # `smtp` queue, so "Outgoing message" would otherwise overtake it on the wire.
+    # Wait for the read receipt to reach Alice before queueing it.
+    alice.wait_for_event(EventType.MSG_READ)
+
     # Send a message and wait until it arrives
     # in order to wait until Bob2 gets the markseen message.
     # This also tests that outgoing messages

--- a/deltachat-rpc-client/tests/test_something.py
+++ b/deltachat-rpc-client/tests/test_something.py
@@ -1005,7 +1005,7 @@ def test_no_markseen_in_team_profile(team_profile, acfactory):
 
     # The MDN is queued in `smtp_mdns`, which is drained only after the regular
     # `smtp` queue, so "Outgoing message" would otherwise overtake it on the wire.
-    # Wait for the read receipt to reach Alice before queueing it.
+    # Wait for the read receipt to reach Alice before queueing "Outgoing message".
     alice.wait_for_event(EventType.MSG_READ)
 
     # Send a message and wait until it arrives

